README.md

```
@@ -1,3 +1,134 @@
+    ５口 口 口 口 口 口 口 口 口
+    ６졸 口 졸 口 졸 口 졸 口 졸
+    ７口 포 口 口 口 口 口 포 口
+    ８口 口 口 口 궁 口 口 口 口
+    ９차 마 상 사 口 사 상 마 차
+    ```
+
+## :rocket: 장기말 이동
+
+- 플레이 시작 문구 출력
+
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)